### PR TITLE
CONTROLLER: Added Home Controller

### PR DESCRIPTION
**Pull request by MahalakshmiMurari**
image

### CODE REVIEW

File: ReserveSpot.Services.Api/Controllers/HomeController.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+namespace ReserveSpot.Services.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class HomeController : RESTFulController
+    {
+        [HttpGet]
+        public ActionResult<string> get() =>
+            Ok("Hurry Up! Reserve Your Spot. We are happy to see you playing sports.");
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/Controllers/HomeController.cs:16]: Attach a screen shot that shows this message.

File: ReserveSpot.Services.Api/Controllers/HomeController.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using Microsoft.AspNetCore.Mvc;
+using RESTFulSense.Controllers;
+
+namespace ReserveSpot.Services.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/Controllers/HomeController.cs:10]: Just for sanity purpose, place the [ApiController] first.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: ReserveSpot.Services.Api/Controllers/HomeController.cs (hunk 1 above)
Comment (MahalakshmiMurari) [reply ReserveSpot.Services.Api/Controllers/HomeController.cs:16]: Home Controller Output

File: ReserveSpot.Services.Api/Controllers/HomeController.cs (hunk 2 above)
Comment (MahalakshmiMurari) [reply ReserveSpot.Services.Api/Controllers/HomeController.cs:10]: Updated the Api Controller first

File: .github/workflows/dotnet.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ jobs:
     - name: Setup Dot Net Version
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 6.0.100
+        dotnet-version: 7.0.0-preview.1
```
Comment (ShriHumrudha) [inline .github/workflows/dotnet.yml]: This should be a separate PR.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: ReserveSpot.Services.Api/Controllers/HomeController.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+namespace ReserveSpot.Services.Api.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class HomeController : RESTFulController
+    {
+        [HttpGet]
+        public ActionResult<string> get() =>
+            Ok("Hurry Up! Reserve Your Spot. We are happy to see you playing sports.");
```
Comment (NaynaMallick) [inline ReserveSpot.Services.Api/Controllers/HomeController.cs:16]: method name should be Get()

File: ReserveSpot.Services.Infrastructure.Build/Program.cs [hunk 5]
```diff
@@ -51,7 +51,7 @@ static void Main(string[] args)
 
                                 TargetDotNetVersion = new TargetDotNetVersion
                                 {
-                                    DotNetVersion = "6.0.100",
+                                    DotNetVersion = "7.0.0-preview.1",
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Infrastructure.Build/Program.cs]: Please do not do this here.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: ReserveSpot.Services.Api/Controllers/HomeController.cs (hunk 4 above)
Comment (MahalakshmiMurari) [reply ReserveSpot.Services.Api/Controllers/HomeController.cs:16]: Method name changed to Get()

File: .github/workflows/dotnet.yml (hunk 3 above)
Comment (MahalakshmiMurari) [reply .github/workflows/dotnet.yml]: Reverted to 6.0.100 . I will do that in a separate PR

File: ReserveSpot.Services.Infrastructure.Build/Program.cs (hunk 5 above)
Comment (MahalakshmiMurari) [reply ReserveSpot.Services.Infrastructure.Build/Program.cs]: Reverted to 6.0.100 . I will do that in a separate PR

**Review by ShriHumrudha — APPROVED** [review]
